### PR TITLE
feat: add Elixir support

### PR DESCRIPTION
**Pull request by asurkov**
based on ExtractorBase from PLpgSQL pull request

### CODE REVIEW

File: src/main/kotlin/app/extractors/PLpgSQLExtractor.kt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// Copyright 2018 Sourcerer Inc. All Rights Reserved.
+// Author: Alexander Surkov (alex@sourcerer.io)
+
+package app.extractors
+
+object PLpgSQLExtractor : ExtractorBase(
```
Comment (anatolystansler) [inline src/main/kotlin/app/extractors/PLpgSQLExtractor.kt]: Can we add PLpgSQL with #323?

Comment (asurkov) [reply src/main/kotlin/app/extractors/PLpgSQLExtractor.kt]: sure, this branch is based on on that branch, as long as that thing is landed, this one will be rebased relative master, thus no merging and stuff

File: src/main/kotlin/app/extractors/ElixirExtractor.kt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+// Copyright 2018 Sourcerer Inc. All Rights Reserved.
+// Author: Alexander Surkov (alex@sourcerer.io)
+
+package app.extractors
+
+object ElixirExtractor : ExtractorBase(
+    language = Lang.ELIXIR,
+    importRegex = Regex("""^\s+(?:use|import|require) ([a-zA-Z_][a-zA-Z0-9_]*)"""),
+    commentRegex = Regex("""#.*$"""))
+{
```
Comment (anatolystansler) [inline src/main/kotlin/app/extractors/ElixirExtractor.kt]: Remove empty body.

File: src/main/kotlin/app/extractors/PLpgSQLExtractor.kt [hunk 3]
```diff
@@ -6,5 +6,4 @@ package app.extractors
 object PlpgsqlExtractor : ExtractorBase(
```
Comment (anatolystansler) [inline src/main/kotlin/app/extractors/PLpgSQLExtractor.kt]: File name violates naming conventions.

**Pull request merged**